### ISSUE TITLE
No way to decline trade/talk about revised offer

### ISSUE THREAD

**Issue by ghost** [dc74089/oneshirt#3]
As it stands, the only options are either accept (which gives the impression you're ready to trade) or close (which doesn't make it clear that they actually saw your message). We need some type of decline message or something where the people can negotiate.

**Comment by dc74089** [comment]
Good point. I think the current flow of trading works, but the verbage needs to be tweaked.

**Comment by dc74089** [comment]
Accidentally closed wrong issue via commit.

**Comment by dc74089** [comment]
As of d4525b922a5b961bc382b9f0888157616202141a, trades can be declined. Still no ability to negotiate.
